src/libraries/System.ComponentModel.TypeConverter/src/System/Timers/TimersDescriptionAttribute.cs

```diff
@@ -23,7 +23,11 @@ public class TimersDescriptionAttribute : DescriptionAttribute
         /// <summary>
         /// Constructs a new localized sys description.
         /// </summary>
-        internal TimersDescriptionAttribute(string description, string defaultValue) : base(SR.GetResourceString(description, defaultValue)) { }
+        internal TimersDescriptionAttribute(string description, string unused) : base(SR.GetResourceString(description))
+        {
+            // Needed for overload resolution
+            System.Diagnostics.Debug.Assert(unused == null);
```

Comment:
Nit: we generally would have a `using System.Diagnostics;` and then just `Debug.Assert(unused == null)` here.